modules/seqexec/server/src/main/scala/seqexec/server/keywords/package.scala

```diff
@@ -275,7 +275,7 @@ package object keywords {
     // Check if there is an error reading a value and if there is a failure
     //  use the default
     def safeValOrDefault: F[A] =
-      v.attempt.map(_.getOrElse(DefaultHeaderValue[A].default))
+      v.handleError(_ => DefaultHeaderValue[A].default)
```

Comment:
@tpolecat `handleError` seems to be correct here but my tests at the console show the line above does the same. Perhaps we could discuss the details

Reply:
Yeah `.attempt` isn't a reliable way to trap errors because following `map/flatMap` may never happen, depending on `F`. If you need to trap or handle errors use the combinators on `ApplicativeError/MonadError`.

Reply:
We can talk about it in the morning.